test/integration/targets/canonical-pep517-self-packaging/minimum-build-constraints.txt

```diff
@@ -0,0 +1,16 @@
+# Lowest supporting Python 3.9 and 3.10:
```

Comment:
Should there be any changes here since stable-2.13 supports Python 3.8 - 3.10?

Reply:
Looking at this closer, it will need changes, since it's not installing `setuptools` on Python 3.8.